oct_converter/readers/binary_structs/fds_binary.py

```diff
@@ -92,26 +196,36 @@
 )
 
 capture_info_02_header = Struct(
-    "x" / Int16un,
-    "zeros" / PaddedString(52, "ascii"),
-    "aquisition_year" / Int16un,
-    "aquisition_month" / Int16un,
-    "aquisition_day" / Int16un,
-    "aquisition_hour" / Int16un,
-    "aquisition_minute" / Int16un,
-    "aquisition_second" / Int16un,
+    "eye" / Int8un,
+    "scan_mode" / Int8un,
+    "session_id" / Int32un,
+    "label" / PaddedString(100, "ascii"),
+    "cap_date" / Int16un[6],
 )
 
+capture_info_header = Struct(
+    "eye" / Int8un,
+    "cap_date" / Int16un[6],
+)
 
 img_trc_02_header = Struct(
     "width" / Int32un,
     "height" / Int32un,
     "bits_per_pixel" / Int32un,
     "num_slices_0x2" / Int32un,
-    "0x1" / Int8un,
+    "format" / Int8un,
     "size" / Int32un,
 )
 
+img_trc_header = Struct(
+    "width" / Int32un,
+    "height" / Int32un,
+    "bits_per_pixel" / Int32un,
+    "size" / Int32un,
+)
+
+# TODO This chunk still needs work.

```

Comment:
Thank you! Figured out today that PARAM_OBS_02 has a chunk size of *either* 6 or 90. I should be able to squeeze a little more info out of this chunk while still on this MR.

Reply:
What's your preference for where this chunk gets handled? My first thought was making a dedicated method to parse this chunk, but it could also be handled with an if statement in `read_any_info_and_make_dict`.

Reply:
i think a dedicated method makes sense, given there is already a precedent of doing it that way for more complicated chunks (like segmentation contours, scan params)

Reply:
Updated, let me know what you think and any changes!